core-bundle/src/Resources/config/services.yml

```diff
@@ -1,3 +1,11 @@
+parameters:
+    contao.content_element.templates: []
+    contao.frontend_module.templates: []
+    contao.content_proxy: Contao\ContentProxy
+    contao.module_proxy: Contao\ModuleProxy
+    contao.content_element_template_prefix: ce_
+    contao.frontend_module_template_prefix: mod_
```

Comment:
Shouldn't the defaults be defined in the `Configuration` class?

Reply:
I don't think so, as this is not part of the bundle configuration.